wiki/Ranking_criteria/osu!taiko/zh.md

```diff
@@ -1,3 +1,8 @@
+---
```

Comment:
`of each included game mode` is not translated into osu!taiko/zh.md (it doesn't make sense), so it may be better not to mark it as outdated?